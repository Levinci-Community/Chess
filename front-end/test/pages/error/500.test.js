import React from "react";
import { render } from "@testing-library/react";
import InternalServerErrorPage from "../../../src/pages/error/500";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders InternalServerErrorPage snapshot", () => {
    const { container } = render(
      <Router>
        <InternalServerErrorPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });