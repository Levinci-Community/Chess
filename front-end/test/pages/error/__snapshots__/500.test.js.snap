// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InternalServerErrorPage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-vl7sen"
  >
    <div
      class="chakra-stack css-1kbzohu"
    >
      <h2
        class="chakra-heading css-zov4ca"
      >
        500 Internal Server Error
      </h2>
      <p
        class="chakra-text css-0"
      >
        Something went wrong on our end. Our team has been notified, and we're working to fix it. Please try again later.
      </p>
    </div>
  </div>
</div>
`;
